### ISSUE TITLE
Bot don't respond to the intro messages of the users in a pool 

### ISSUE THREAD

**Issue by KohliAnuj** [liddiard/slack-meetups#14]
Hi- I was trying to set up the project locally. While the bot was able to dm each user to provide their intros, nothing happens after the user provides the same. The bot doesn't respond and in the backend, the user is not marked available. What could be the issue in the setup? Is it a oAuth permission issue- I have given all admin permission scope though to the application. Thanks

**Comment by KohliAnuj** [comment]
@liddiard, it would be great if you could help me with my issue. Im a bit stuck . Thanks

**Comment by liddiard** [comment]
Hi @KohliAnuj, are you using the `master` branch of the project ([webhook-based](https://api.slack.com/events-api)), or the `rtm` branch ([socket-based](https://api.slack.com/rtm))? 

If you're using the master branch, have you:

1. Created an app for the bot at https://api.slack.com/apps/
2. Enabled "Interactivity & Shortcuts" with a URL that's `<your base URL>/slack/action/` and "Event Subscriptions" with a URL that's `<your base URL>/slack/message/` for the same app? (see screenshots below for an example)

<img width="1434" alt="Screen Shot 2020-07-09 at 9 51 44 AM" src="https://user-images.githubusercontent.com/3317632/87068846-0aaf3d80-c1cb-11ea-87cb-36e370cb42f8.png">
<img width="1434" alt="Screen Shot 2020-07-09 at 9 52 42 AM" src="https://user-images.githubusercontent.com/3317632/87068854-0daa2e00-c1cb-11ea-85af-86c6d097c544.png">

Basically, Slack needs to have endpoints that are accessible on the public internet to send HTTP requests with the messages that the Meetups bot receives.

If you're running the Django server on your local computer, you'll probably need to use something like [ngrok](https://ngrok.com/) to expose the it to the public internet. If you have the project deployed on a public web server, then you can use that server's hostname for the "base URL."

Feel free to let me know if you have followup questions. The setup instructions don't explain how to set up the connection between Slack and the server, and I should probably add that.

**Comment by KohliAnuj** [comment]
Thanks- it worked! Yes it would be helpful to others if you could add the above setup as well to readme

@liddiard wanted to ask another thing- right now GET <base_url>/stats/<channel_name> is showing not found. My channel name is #meetups. How should I put that in the channel name? I tried with meetups but it didn't work. 

Thanks!

**Comment by KohliAnuj** [comment]
@liddiard it would be great if you could share how to fix the above query! thanks

**Comment by liddiard** [comment]
Hi @KohliAnuj, so sorry for the late reply, I need to check these notifications more often! Hope you were able to figure out the issue, but if not could you share a screenshot of the values you have for the "pool" fields in the admin and the URL you're trying to access? And just to be sure, the error it's giving you is a 404 when you try to access the "stats" page?

Here's an example of a configuration I have working:

## Pool config in admin

<img width="692" alt="Screen Shot 2020-08-12 at 10 06 04 AM" src="https://user-images.githubusercontent.com/3317632/90045445-e8a25280-dc83-11ea-9eab-37300adc2961.png">

## Corresponding stats page URL

<img width="494" alt="Screen Shot 2020-08-12 at 10 06 47 AM" src="https://user-images.githubusercontent.com/3317632/90045493-f5bf4180-dc83-11ea-9503-6e30824f5189.png">

Anyway, let me know if you're still having this issue and I'll try to help. Made sure I'm getting email notifications for GitHub issues now so I should see these messages sooner.

**Comment by liddiard** [comment]
Also, I haven't had time to make a proper instructions writeup, but I [added some info to the readme about my setup](https://github.com/liddiard/slack-meetups/commit/0d8c42c2fdd659cca72f12cc2ed2ad3cd8a86b16) if you may find that useful.
